src/components/player-controls.ts

```diff
@@ -22,6 +22,14 @@ class PlayerControls extends LitElement {
     this.mediaControlService = this.store.mediaControlService;
 
     const noUpDown = !!this.config.showVolumeUpAndDownButtons && nothing;
+    const hideNextTrack = this.config.hidePlayerControlNextTrackButton || nothing;
+    const hidePrevTrack = this.config.hidePlayerControlPrevTrackButton || nothing;
+    const hideRepeat = this.config.hidePlayerControlRepeatButton || nothing;
+    const hideShuffle = this.config.hidePlayerControlShuffleButton || nothing;
+
+    const supportsTurnOn = (this.activePlayer.attributes.supported_features || 0) & TURN_ON;
+    const showPowerButton = supportsTurnOn && this.config.showPlayerControlPowerButton && nothing;
```

Comment:
Default should be to show the power button if it is supported. So please change it to `hidePlayerControlPowerButton`

Reply:
I changed the `showPlayerControlPowerButton` to `hidePlayerControlPowerButton` and committed the changes to the featurePlayerControlsHideShow branch.  Do I need to submit another PR after doing that?